### PR TITLE
surf-1206-deprecate-choice-tile-custom-element

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: surf-1206

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

File: docs/components/choice-tiles/index.html [hunk 1]
```diff
@@ -54,14 +50,14 @@ <h3>Basic Choice Tile</h3>
             :checked="isChecked"
             :disabled="isDisabled"
             :invalid="isInvalid" />
-          <hx-tile :class="tileClasses">
+          <div class="hxTile" :class="tileClasses">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/index.html]: Check to see if `class="hxTile"` needed.

File: docs/components/choice-tiles/test.html [hunk 2]
```diff
@@ -86,40 +86,40 @@ <h2>Default</h2>
...
+          <header>Default (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input:focus -->
       <label class="hxChoice">
         <input type="radio" name="default2" class="pseudo-focus" />
-        <hx-tile>
+        <div class="hxTile">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: Check if `.pseudo-focus` should be on the tile like hover

File: docs/components/choice-tiles/test.html [hunk 3]
```diff
@@ -131,40 +131,40 @@ <h2>Subdued</h2>
...
+          <header>Subdued (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input:focus + .hxSubdued -->
       <label class="hxChoice">
         <input type="radio" name="subdued2" class="pseudo-focus" />
-        <hx-tile class="hxSubdued">
+        <div class="hxTile hxSubdued">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: Ditto

File: docs/components/choice-tiles/test.html [hunk 4]
```diff
@@ -176,40 +176,40 @@ <h2>Checked</h2>
...
+          <header>Checked (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input:checked:focus -->
       <label class="hxChoice">
         <input type="radio" name="checked2" checked class="pseudo-focus" />
-        <hx-tile>
+        <div class="hxTile">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: ditto

File: docs/components/choice-tiles/test.html [hunk 5]
```diff
@@ -222,40 +222,40 @@ <h2>Checked + Subdued</h2>
...
+          <header>Checked + Subdued (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input:checked:focus + .hxSubdued -->
       <label class="hxChoice">
         <input type="radio" name="checkSub2" checked class="pseudo-focus" />
-        <hx-tile class="hxSubdued">
+        <div class="hxTile hxSubdued">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: ditto

File: docs/components/choice-tiles/test.html [hunk 6]
```diff
@@ -267,40 +267,40 @@ <h2>Invalid</h2>
...
+          <header>Invalid (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input[invalid]:focus -->
       <label class="hxChoice">
         <input type="radio" name="invalid2" invalid class="pseudo-focus" />
-        <hx-tile>
+        <div class="hxTile">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: ditto

File: docs/components/choice-tiles/test.html [hunk 7]
```diff
@@ -313,40 +313,40 @@ <h2>Invalid + Subdued</h2>
...
+          <header>Invalid + Subdued (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input[invalid]:focus + .hxSubdued -->
       <label class="hxChoice">
         <input type="radio" name="invalidSub2" invalid class="pseudo-focus" />
-        <hx-tile class="hxSubdued">
+        <div class="hxTile hxSubdued">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: ditto

File: docs/components/choice-tiles/test.html [hunk 8]
```diff
@@ -390,40 +390,40 @@ <h2>Invalid + Checked</h2>
...
+          <header>Invalid + Checked (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input[invalid]:checked:focus -->
       <label class="hxChoice">
         <input type="radio" name="invalidCheck3" invalid checked class="pseudo-focus" />
-        <hx-tile>
+        <div class="hxTile">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: ditto

File: docs/components/choice-tiles/test.html [hunk 9]
```diff
@@ -436,40 +436,40 @@ <h2>Invalid + Checked + Subdued</h2>
...
+          <header>Invalid + Checked + Subdued (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input[invalid]:checked:focus + .hxSubdued -->
       <label class="hxChoice">
         <input type="radio" name="invChkSub2" invalid checked class="pseudo-focus" />
-        <hx-tile class="hxSubdued">
+        <div class="hxTile hxSubdued">
```
Comment (nimbinatus) [inline docs/components/choice-tiles/test.html]: ditto

File: src/helix-ui/styles/components/choice-tiles.less [hunk 10]
```diff
@@ -5,6 +5,202 @@
...
+  p {
+    color: inherit;
+  }
+
+  hx-icon {
+    color: @gray-500;
+  }
+}
+
+// hx-tile class
```
Comment (nimbinatus) [inline src/helix-ui/styles/components/choice-tiles.less]: Update comment

File: docs/components/choice-tiles/index.html (hunk 1 above)
Comment (cathysiller) [reply docs/components/choice-tiles/index.html]: not needed, removing

File: docs/components/choice-tiles/test.html (hunk 2 above)
Comment (cathysiller) [reply docs/components/choice-tiles/test.html]: the input gets the `:focus` so when a user tab to a tile, the `input` is focused, not the div, double checked with @CITguy on this one

Comment (CITguy) [reply docs/components/choice-tiles/test.html]: Yup, focus applied to input, not div. Marking comments resolved.

File: src/helix-ui/styles/elements/hx-tile.less [hunk 11]
```diff
@@ -7,15 +7,23 @@ hx-tile {
   text-align: center;
   width: 100%;
 
-  hx-tile-title {
```
Comment (CITguy) [inline src/helix-ui/styles/elements/hx-tile.less]: Please don't remove the old CSS just yet. We want the deprecation to be backward-compatible.

Make sure to move/copy the old definitions to a section at the bottom of this file for deprecation.

File: src/helix-ui/styles/components/choice-tiles.less [hunk 12]
```diff
@@ -67,7 +67,7 @@ label.hxChoice > input:checked + hx-tile {
   border-color: @cyan-900;
   color: @cyan-900;
 
-  hx-tile-description {
```
Comment (CITguy) [inline src/helix-ui/styles/components/choice-tiles.less]: Make sure to move/copy the old definitions to a section at the bottom of this file for deprecation.

**Pull request merged**